core/domain/topic_domain.py

```diff
@@ -505,7 +512,9 @@ def require_valid_abbreviated_name(cls, name):
             raise utils.ValidationError(
                 'Abbreviated name field should not be empty.')
 
-        if len(name) > 12:
+        if (
+                len(name) >
+                android_checks.TopicConstants.MAX_CHARS_IN_ABBREVIATED_TOPIC_NAME):  # pylint: disable=line-too-long
```

Comment:
Same thing, here and elsewhere, avoid pylint-disable.

Reply:
Done